perfschema/init.go

```diff
@@ -210,9 +218,19 @@ func (ps *perfSchema) buildTables() error {
 			c.ID = autoid.GenLocalSchemaID()
 		}
 		alloc := autoid.NewMemoryAllocator(dbID)
-		tbl, err := createMemoryTable(meta, alloc)
-		if err != nil {
-			return errors.Trace(err)
+
+		var tbl table.Table
+		var err error
+		switch name {
+		case TableStmtsCurrent, TablePreparedStmtsInstances, TableTransCurrent, TableStagesCurrent:
+			tbl = createBoundedTable(meta, alloc, currentElemMax)
+		case TableStmtsHistory, TableStmtsHistoryLong, TableTransHistory, TableTransHistoryLong, TableStagesHistory, TableStagesHistoryLong:
+			tbl = createBoundedTable(meta, alloc, historyElemMax)
+		default:
```

Comment:
`setup_xxx` tables are used to store configuration information, it's similar to other schema tables, so I would like these tables to use MemoryTable.